Floofbot/Modules/NicknameAlert.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Discord.Commands;
+using Floofbot.Services.Repository;
+using Floofbot.Services.Repository.Models;
+using Serilog;
+using System;
+using System.Threading.Tasks;
+
+namespace Floofbot.Modules
+{
+    public class NicknameAlert : ModuleBase<SocketCommandContext>
```

Comment:
I'd honestly prefer a naming convention of XModule or XCommands for these module classes, but we can address this in a future PR.